SeaPublicWebsite.Data/Migrations/SeaDbContextModelSnapshot.cs

```diff
@@ -281,7 +284,8 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                 {
                     b.HasOne("SeaPublicWebsite.BusinessLogic.Models.PropertyData", null)
                         .WithOne("UneditedData")
-                        .HasForeignKey("SeaPublicWebsite.BusinessLogic.Models.PropertyData", "EditedDataId");
+                        .HasForeignKey("SeaPublicWebsite.BusinessLogic.Models.PropertyData", "EditedDataId")
+                        .OnDelete(DeleteBehavior.Cascade);

```

Comment:
Why is this change in this commit? It should already be part of the snapshot file?

Reply:
I'm not sure... unless it wasn't before?